modules/3d-tiles/src/parsers/helpers/parse-3d-tile-gltf-view.js

```diff
@@ -17,75 +17,75 @@ export const GLTF_FORMAT = {
 };
 
 export function parse3DTileGLTFViewSync(tile, arrayBuffer, byteOffset) {
+  // Set flags
+  // glTF models need to be rotated from Y to Z up
+  // https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#y-up-to-z-up
+  tile.rotateYtoZ = true;
+
   // Assume glTF consumes rest of tile
   const gltfByteLength = tile.byteOffset + tile.byteLength - byteOffset;
   if (gltfByteLength === 0) {
     throw new Error('glTF byte length must be greater than 0.');
   }
 
   // TODO - We can avoid copy if already 4-byte aligned...
-  // if (byteOffset % 4 === 0) {
-  //   tile.gltfArrayBuffer = arrayBuffer;
-  //   tile.gltfByteOffset = byteOffset;
-  //   tile.gltfByteLength = gltfByteLength;
-  // } else {
-  // Create a copy of the glb so that it is 4-byte aligned
-  // eslint-disable-next-line
-  // console.warn(`${tile.type}: embedded glb is not aligned to a 4-byte boundary.`);
-  const subArray = new Uint8Array(arrayBuffer).subarray(byteOffset, byteOffset + gltfByteLength);
-  const arrayCopy = new Uint8Array(subArray);
-  tile.gltfArrayBuffer = arrayCopy.buffer;
+  // However the rest of the code may not be able to accept byteOffsets, so copy anyway
+  tile.gltfArrayBuffer = copyArrayBuffer(arrayBuffer, byteOffset, gltfByteLength);
   tile.gltfByteOffset = 0;
   tile.gltfByteLength = gltfByteLength;
-  // }
 
-  // glTF models need to be rotated from Y to Z up
-  // https://github.com/AnalyticalGraphicsInc/3d-tiles/tree/master/specification#y-up-to-z-up
-  tile.rotateYtoZ = true;
+  if (byteOffset % 4 === 0) {
+    // tile.gltfArrayBuffer = arrayBuffer;
```

Comment:
remove?

Reply:
Added comments explaining why this is still there.